packages/ua-devtools/src/oapp/types.ts

```diff
@@ -14,7 +14,7 @@ import type {
 import { ExecutorOptionType } from '@layerzerolabs/lz-v2-utilities'
 
 export interface IOApp extends IOmniSDK {
-    getEndpointSDK(): Promise<IEndpoint>
```

Comment:
I think we should keep this one as-is for now, until we decide what to do about interoperability

Reply:
Done.